### DISCUSSION TITLE
Cannot find bitbake command inside Yocto

### DISCUSSION THREAD

**Discussion by maitreyakale** [polarfire-soc/polarfire-soc-documentation#310] [Linux build systems for PolarFire SoC]
We are trying to implement Yocto Project using Icicle Kit! we followed all the steps that given in the https://github.com/polarfire-soc/meta-polarfire-soc-yocto-bsp#Dependencies,but during "**MACHINE=icicle-kit-es bitbake mpfs-dev-cli**" implementation of this command following error i.e. **"bitbake: command not found"** is occured ! please give solution onto this.....

**Comment by vfalanis** [comment]
Hi @maitreyakale,

Once you installed all the dependencies in your host machine, you need to follow the [Build Instructions](https://github.com/polarfire-soc/meta-polarfire-soc-yocto-bsp#build-instructions) section of the PolarFire SoC Yocto BSP README.

Based on the information provided, it looks like you haven't setup the Bitbake environment:
```
. ./meta-polarfire-soc-yocto-bsp/polarfire-soc_yocto_setup.sh
```

I would suggest to follow the Build Instructions section in the link provided above, that should solve the issue you are facing.

Thanks
